<<<<<<< HEAD
[ERROR][conn.go:30]2022/04/02 08:06:44 [the database connect fail]
[INFO][define.go:25]2022/04/02 08:06:44 [1212]
[ERROR][conn.go:32]2022/04/02 08:07:51 [the database connect success]
[INFO][define.go:25]2022/04/02 08:07:51 [1212]
[ERROR][deleteService.go:68]2022/04/02 08:48:53 [remove /resources/6915822239413833728.png: The system cannot find the file specified.]
[ERROR][conn.go:30]2022/04/02 12:37:16 [the database connect fail]
[INFO][define.go:25]2022/04/02 12:37:16 [1212]
[ERROR][conn.go:32]2022/04/02 12:37:23 [the database connect success]
[INFO][define.go:25]2022/04/02 12:37:23 [1212]
[ERROR][conn.go:32]2022/04/02 13:44:03 [the database connect success]
[INFO][define.go:25]2022/04/02 13:44:03 [1212]
[ERROR][conn.go:32]2022/04/02 14:07:57 [the database connect success]
[INFO][define.go:25]2022/04/02 14:07:57 [1212]
[ERROR][conn.go:32]2022/04/02 14:21:25 [the database connect success]
[INFO][define.go:25]2022/04/02 14:21:25 [1212]
[ERROR][conn.go:32]2022/04/02 14:22:24 [the database connect success]
[INFO][define.go:25]2022/04/02 14:22:24 [1212]
=======
[ERROR][conn.go:32]2022/04/02 00:18:05 [the database connect success]
[INFO][define.go:25]2022/04/02 00:18:05 [1212]
[ERROR][conn.go:32]2022/04/02 08:42:02 [the database connect success]
[INFO][define.go:25]2022/04/02 08:42:02 [1212]
[ERROR][GetActivityById.go:29]2022/04/02 09:15:30 [no rows in result set]
[ERROR][GetActivityById.go:29]2022/04/02 09:17:21 [no rows in result set]
[ERROR][GetActivityById.go:29]2022/04/02 09:24:58 [no rows in result set]
[ERROR][conn.go:32]2022/04/02 10:01:09 [the database connect success]
[INFO][define.go:25]2022/04/02 10:01:09 [1212]
<<<<<<< HEAD
<<<<<<< HEAD
[ERROR][conn.go:32]2022/04/02 17:56:22 [the database connect success]
[INFO][define.go:25]2022/04/02 17:56:22 [1212]
[ERROR][conn.go:32]2022/04/02 17:57:15 [the database connect success]
[INFO][define.go:25]2022/04/02 17:57:15 [1212]
[ERROR][conn.go:32]2022/04/02 18:01:21 [the database connect success]
[INFO][define.go:25]2022/04/02 18:01:21 [1212]
=======
[ERROR][conn.go:32]2022/04/02 16:13:50 [the database connect success]
[INFO][define.go:25]2022/04/02 16:13:50 [1212]
[ERROR][conn.go:32]2022/04/02 16:16:15 [the database connect success]
[INFO][define.go:25]2022/04/02 16:16:15 [1212]
[ERROR][conn.go:32]2022/04/02 16:17:47 [the database connect success]
[INFO][define.go:25]2022/04/02 16:17:47 [1212]
[ERROR][conn.go:32]2022/04/02 16:28:47 [the database connect success]
[INFO][define.go:25]2022/04/02 16:28:47 [1212]
[ERROR][conn.go:32]2022/04/02 16:34:02 [the database connect success]
[INFO][define.go:25]2022/04/02 16:34:02 [1212]
[ERROR][conn.go:32]2022/04/02 16:35:18 [the database connect success]
[INFO][define.go:25]2022/04/02 16:35:18 [1212]
[ERROR][login.go:57]2022/04/02 16:37:52 [no rows in result set]
[ERROR][conn.go:32]2022/04/02 16:39:25 [the database connect success]
[INFO][define.go:25]2022/04/02 16:39:25 [1212]
<<<<<<< HEAD
>>>>>>> BlackHat
=======
[ERROR][conn.go:32]2022/04/02 16:15:50 [the database connect success]
[INFO][define.go:25]2022/04/02 16:15:50 [1212]
[ERROR][conn.go:32]2022/04/02 16:16:04 [the database connect success]
[INFO][define.go:25]2022/04/02 16:16:04 [1212]
[ERROR][conn.go:32]2022/04/02 16:18:08 [the database connect success]
[INFO][define.go:25]2022/04/02 16:18:08 [1212]
[ERROR][conn.go:32]2022/04/02 16:18:24 [the database connect success]
[INFO][define.go:25]2022/04/02 16:18:24 [1212]
[ERROR][conn.go:32]2022/04/02 16:26:21 [the database connect success]
[INFO][define.go:25]2022/04/02 16:26:21 [1212]
[ERROR][conn.go:32]2022/04/02 16:33:58 [the database connect success]
[INFO][define.go:25]2022/04/02 16:33:58 [1212]
>>>>>>> zhangyouwei
<<<<<<< HEAD
[ERROR][conn.go:32]2022/04/02 19:35:02 [the database connect success]
[INFO][define.go:25]2022/04/02 19:35:02 [1212]
[ERROR][deleteService.go:68]2022/04/02 19:40:03 [remove /resources/6915985539778875392.jpg: The system cannot find the file specified.]
[ERROR][login.go:57]2022/04/02 20:10:14 [no rows in result set]
[ERROR][login.go:57]2022/04/02 22:28:39 [no rows in result set]
=======
[ERROR][conn.go:32]2022/04/02 16:41:53 [the database connect success]
[INFO][define.go:25]2022/04/02 16:41:53 [1212]
[ERROR][deleteService.go:68]2022/04/02 16:56:53 [remove /resources/6915944570161336320.png: The system cannot find the file specified.]
[ERROR][conn.go:32]2022/04/02 19:30:42 [the database connect success]
[INFO][define.go:25]2022/04/02 19:30:42 [1212]
[ERROR][conn.go:32]2022/04/02 19:35:35 [the database connect success]
[INFO][define.go:25]2022/04/02 19:35:35 [1212]
[ERROR][conn.go:32]2022/04/02 19:39:38 [the database connect success]
[INFO][define.go:25]2022/04/02 19:39:38 [1212]
[ERROR][uploadService.go:124]2022/04/02 19:48:11 [ERROR: duplicate key value violates unique constraint "t_file_mapper_pkey" (SQLSTATE 23505)]
[ERROR][conn.go:32]2022/04/02 20:24:50 [the database connect success]
[INFO][define.go:25]2022/04/02 20:24:50 [1212]
[ERROR][uploadService.go:124]2022/04/02 20:25:33 [ERROR: duplicate key value violates unique constraint "t_file_mapper_pkey" (SQLSTATE 23505)]
[ERROR][deleteService.go:61]2022/04/02 20:28:59 [write failed: write tcp 172.29.14.179:64551->172.22.72.229:16900: wsasend: An existing connection was forcibly closed by the remote host.]
[ERROR][uploadService.go:124]2022/04/02 20:31:14 [ERROR: duplicate key value violates unique constraint "t_file_mapper_pkey" (SQLSTATE 23505)]
[ERROR][conn.go:30]2022/04/02 22:04:00 [the database connect fail]
[INFO][define.go:25]2022/04/02 22:04:00 [1212]
[ERROR][conn.go:32]2022/04/02 22:05:45 [the database connect success]
[INFO][define.go:25]2022/04/02 22:05:45 [1212]
[ERROR][conn.go:32]2022/04/02 22:11:23 [the database connect success]
[INFO][define.go:25]2022/04/02 22:11:23 [1212]
>>>>>>> BlackHat
[ERROR][conn.go:32]2022/04/02 22:45:18 [the database connect success]
[INFO][define.go:25]2022/04/02 22:45:18 [1212]
[ERROR][conn.go:32]2022/04/02 23:02:04 [the database connect success]
[INFO][define.go:25]2022/04/02 23:02:04 [1212]
[ERROR][conn.go:32]2022/04/02 23:02:47 [the database connect success]
[INFO][define.go:25]2022/04/02 23:02:47 [1212]
[ERROR][conn.go:32]2022/04/02 23:11:39 [the database connect success]
[INFO][define.go:25]2022/04/02 23:11:39 [1212]
[ERROR][deleteService.go:61]2022/04/02 23:15:39 [write failed: write tcp 2.0.1.61:57831->172.22.72.229:16900: wsasend: An existing connection was forcibly closed by the remote host.]
[ERROR][conn.go:32]2022/04/02 23:38:00 [the database connect success]
[INFO][define.go:25]2022/04/02 23:38:00 [1212]
=======
>>>>>>> bf9ebba91c62f3aeb28bd12b7ed24defe026d2bc
[ERROR][conn.go:32]2022/04/02 20:06:33 [the database connect success]
[INFO][define.go:25]2022/04/02 20:06:33 [1212]
[ERROR][conn.go:32]2022/04/02 20:06:56 [the database connect success]
[INFO][define.go:25]2022/04/02 20:06:56 [1212]
[ERROR][conn.go:32]2022/04/02 20:17:41 [the database connect success]
[INFO][define.go:25]2022/04/02 20:17:41 [1212]
>>>>>>> chenwenhan
